### PR TITLE
delete cell metadata in bigquery (for SCP-2039 and SCP-2043)

### PR DESCRIPTION
**Pull request by skiggety**

### CODE REVIEW

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -1,3 +1,5 @@
+require "google/cloud/bigquery"
```
Comment (bistline) [inline app/controllers/application_controller.rb]: Do we need this require statement here?  I though including it in the Gemfile was sufficient.

File: test/integration/study_creation_test.rb [hunk 2]
```diff
@@ -108,23 +118,48 @@ class StudyCreationTest < ActionDispatch::IntegrationTest
...
-    study.reload
     gene_count = study.gene_count
     cluster_count = study.cluster_groups.size
     metadata_count = study.cell_metadata.size
     cluster_annot_count = study.cluster_annotation_count
     study_file_count = study.study_files.non_primary_data.size
     share_count = study.study_shares.size
 
-    assert_equal 19, gene_count, "did not find correct number of genes"
+    # I don't need this, perhaps, because this happens a little later: assert_equal 19, gene_count, "did not find correct number of genes"
```
Comment (bistline) [inline test/integration/study_creation_test.rb]: I would remove this if we in fact don't need it anymore.

File: test/integration/study_creation_test.rb [hunk 3]
```diff
@@ -108,23 +118,48 @@ class StudyCreationTest < ActionDispatch::IntegrationTest
...
-    assert_equal 3, metadata_count, "did not find correct number of metadata objects"
+    assert_equal 22, metadata_count, "did not find correct number of metadata objects"
     assert_equal 2, cluster_annot_count, "did not find correct number of cluster annotations"
     assert_equal 3, study_file_count, "did not find correct number of study files"
     assert_equal 1, share_count, "did not find correct number of study shares"
 
+    assert_equal initial_bq_row_count + 30, get_bq_row_count(bq_dataset, study)
+
+    # request delete
+    study.study_files.each do |sf|
```
Comment (bistline) [inline test/integration/study_creation_test.rb]: Do we need to delete all the study files here?  Don't we just want to delete the metadata file and confirm that it cleans up?  After looking at this, this almost seems like this portion (the delete and BQ validation) should be its own  validation test that would run after this test completes and ingests data correctly.

Comment (skiggety) [reply test/integration/study_creation_test.rb]: I'll change this. I had been thinking about checking that all the files got deleted while I was at it, but ended up not wanting to more wait time to the test.

File: app/controllers/application_controller.rb (hunk 1 above)
Comment (skiggety) [reply app/controllers/application_controller.rb]: Whoops, should have just done it in the gemfile...

File: test/integration/study_creation_test.rb (hunk 2 above)
Comment (skiggety) [reply test/integration/study_creation_test.rb]: all set

File: test/integration/study_creation_test.rb (hunk 3 above)
Comment (skiggety) [reply test/integration/study_creation_test.rb]: all set

File: app/controllers/application_controller.rb (hunk 1 above)
Comment (skiggety) [reply app/controllers/application_controller.rb]: all set now

**Comment by skiggety** [comment]
The test that I worked on for this has exposed an intermittent issue, captured in SCP-2064. It's not an issue with this code or functionality, but we may want to **delay merging this** for now nevertheless so we don't have travis intermittently failing for any branch.

**Comment by bistline** [comment]
After speaking with the OLS team, we've discovered the issue is likely on our end - too many concurrent requests that flood the outgoing TCP/IP socket and cause some to time out or get refused.  We can address this in the ingest pipeline codebase with sleeps and exponential backoffs.

As a result, there isn't anything wrong with this branch, so I'm going to go ahead and merge it.

**Pull request merged**